features/functions/create_data.rb

```diff
@@ -3,8 +3,41 @@ def generate_title_number
   title_number.generate
 end
 
-def create_deed_hash(title_number = generate_title_number)
```

Comment:
It is replaced at the bottom - due a "basic deed" now needing the borrower parts also.